test/e2e-common.sh

```diff
@@ -79,6 +79,48 @@ function show_failure() {
     exit 1
 
 }
+
+function pvc_git_clone() {
+    local maxloop=30 # 5 minutes max
+    local pvc_name=$1
+    local remote=$2
+
+    pwd
+    kubectl -n ${tns} apply -f git/git-clone.yaml
+    cat <<EOF | kubectl -n ${tns} apply -f -
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: git-clone-${pvc_name}
+spec:
+  taskRef:
+    name: git-clone
+  params:
+  - name: url
+    value: ${remote}
+  workspaces:
+  - name: output
+    persistentVolumeClaim:
+      claimName: ${pvc_name}
+EOF
+
+    local cnt=0
+    while true;do
+        [[ ${cnt} == ${maxloop} ]] && show_failure ${testname} ${tns}
+
+        status=$(kubectl get -n ${tns} tr --output=jsonpath='{.items[*].status.conditions[*].status}')
+        reason=$(kubectl get -n ${tns} tr --output=jsonpath='{.items[*].status.conditions[*].reason}')
+        [[ ${status} == *ERROR || ${reason} == *Failed || ${reason} == CouldntGetTask ]] && show_failure ${testname} ${tns}
+        [[ ${status} == True ]] && {
+            echo -n "SUCCESS: ${testname} taskrun has successfully executed: " ;
+            kubectl get pod -o name -n ${tns}|xargs kubectl logs --all-containers -n ${tns}|tail -1
```

Comment:
I know I wrote that but i think it's useless... it show just the same string like "Task Completed"